modules/edit-modes/src/lib/draw-polygon-mode.ts

```diff
@@ -119,9 +119,21 @@ export class DrawPolygonMode extends GeoJsonEditMode {
         coordinates: [[...clickSequence, clickSequence[0]]],
       };
 
+      const editAction = this.getAddFeatureOrBooleanPolygonAction(polygonToAdd, props);
+
+      if (
```

Comment:
This fixes If first editHandle was clicked and last line being created would overlapp